packages/button/button.scss

```diff
@@ -100,6 +93,10 @@ $button-wrapper-animation-safe-area: 10px;
     }
 }
 
+a.jkl-button {
+    text-decoration: none;
+}
+
```

Comment:
Dette trenger vel ikke å være en egen selector? Kan fint sette `text-decoration` på knappe-knappene også 😊

Reply:
Ønsket å kommunisere at det er en reset for `a`